src/store/ChartTableStore.js

```diff
@@ -42,8 +42,8 @@ export default class ChartTableStore {
     @action.bound updateTableData({ DT, Open, High, Low, Close }) {
         this.isTick = this.mainStore.timeperiod.timeUnit === 'tick';
 
-        const year = DT.getUTCFullYear();
-        const month = DT.getUTCMonth() + 1;
+        const year = DT.getFullYear();
+        const month = DT.getMonth() + 1;
         const day = DT.getUTCDate();
```

Comment:
as mentioned earlier, this should also be `getDate()` because it is already in UTC time